includes/abstracts/abstract-wc-widget.php

```diff
@@ -372,7 +373,7 @@ protected function get_current_page_url() {
 		}
 
 		// All current filters.
-		if ( $_chosen_attributes = WC_Query::get_layered_nav_chosen_attributes() ) { // phpcs:ignore Squiz.PHP.DisallowMultipleAssignments.Found, WordPress.CodeAnalysis.AssignmentInCondition.Found
+		if ( $_chosen_attributes = WC_Query::get_layered_nav_chosen_attributes() ) { // phpcs:ignore Squiz.PHP.DisallowMultipleAssignments.FoundInControlStructure, WordPress.CodeAnalysis.AssignmentInCondition.Found
```

Comment:
Updated as the pre-commit hook was failing.

<img width="746" alt="Screenshot 2021-05-31 at 12 23 29" src="https://user-images.githubusercontent.com/1562646/120181381-456bfb00-c20d-11eb-88ab-71c17cd5d6f7.png">